test/env/docker.sh

```diff
@@ -276,20 +276,20 @@ fi
 
 if [ "$SUNDIALS_PRECISION" == "double" ]; then
     export SUNDIALS_SUPERLU_DIST=ON
-    export SUPERLU_DIST_ROOT=/opt/view
+    export SUPERLU_DIST_ROOT=/opt/view/superlu-dist
     export SUPERLU_DIST_INCLUDE_DIR="${SUPERLU_DIST_ROOT}/include"
     export SUPERLU_DIST_LIBRARY_DIR="${SUPERLU_DIST_ROOT}/lib"
 
     # built with 32-bit blas
-    export BLAS_ROOT=/opt/view
+    export BLAS_ROOT=/opt/view/superlu-dist
```

Comment:
I think this and and lines below are the causing the CI failures
```suggestion
    export BLAS_ROOT=/opt/view
```